MHRA/products
# Render a cookie policy

### Name of the story
#### Cookie Banner (GDPR)

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/recqg2COfPJVO6YnV?blocks=hide)

![](https://media.giphy.com/media/xT8qB99xFAXFlyAf6M/giphy.gif)

### Acceptance Criteria

- [ ] A cookie banner should display at the top of the page as per [design](https://zpl.io/adYEDMn)
- [ ] If a user accepts the use of cookies, the banner should not render
- [ ]  the browser should remember users input

### Testing information

- Open the site in the browser of your preference and you should see the cookie policy. 
- Click on accept cookie button and the banner should disappear
- In the same browser open a new tap and go to the site and you shouldn't see a cookie policy banner

### Pre-flight Checklist

- [x] UXD Reviewed

### Pre-merge Checklist
